code401challenges/src/main/java/code401challenges/utilities/AnimalShelter.java

```diff
@@ -0,0 +1,27 @@
+package code401challenges.utilities;
+
+import java.util.Stack;
+
+public class AnimalShelter<T> {
+    private Stack<T> stack1;
+    private Stack<T> stack2;
```

Comment:
I like using two data structures, but I recommend using FIFO data structures for this challenge, it will solve the problem of what animal has been in the shelter the longest.